Source/WTF/wtf/text/StringView.h

```diff
@@ -1250,13 +1250,13 @@ inline size_t findIgnoringASCIICase(StringView source, StringView stringToFind,
 
     if (source.is8Bit()) {
         if (stringToFind.is8Bit())
-            return findIgnoringASCIICase(source.span8().first(searchLength), stringToFind.span8(), static_cast<size_t>(start));
-        return findIgnoringASCIICase(source.span8().first(searchLength), stringToFind.span16(), static_cast<size_t>(start));
+            return findIgnoringASCIICase(source.span8(), stringToFind.span8(), static_cast<size_t>(start));
+        return findIgnoringASCIICase(source.span8(), stringToFind.span16(), static_cast<size_t>(start));
     }
 
     if (stringToFind.is8Bit())
-        return findIgnoringASCIICase(source.span16().first(searchLength), stringToFind.span8(), static_cast<size_t>(start));
-    return findIgnoringASCIICase(source.span16().first(searchLength), stringToFind.span16(), static_cast<size_t>(start));
+        return findIgnoringASCIICase(source.span16(), stringToFind.span8(), static_cast<size_t>(start));
+    return findIgnoringASCIICase(source.span16(), stringToFind.span16(), static_cast<size_t>(start));
```

Comment:
I don't understand the purpose of these two calls to `static_cast`. Should just pass `start`.